src/pages/about.tsx

```diff
@@ -14,25 +14,61 @@ const TeamMemberCard: React.FC<TeamMember> = ({ name, description, profileImg })
   </Card>
 );
 
-const AboutPage: React.FC<PageProps> = () => (
-  <Layout>
-    <div className='max-w-5xl mx-auto px-8 flex flex-col py-8'>
-      <h1 className='text-4xl font-bold'>About Us</h1>
-      <p className='mt-4'>
-        BUGS is the premier open source club at NYU; we advocate for the ideals of open source via
-        first-hand experience of students having an exciting and inclusive environment for them to
-        develop their own projects, work together with other members on larger projects, and take
-        their skills to contribute to pre-existing open source projects.
-      </p>
-      <h2 className='mt-12 text-4xl font-bold'>Executive Board</h2>
-      <div className='mt-4 mb-16 grid gap-8 grid-cols-1 sm:grid-cols-2 lg:grid-cols-3'>
-        {team.map((teamMember) => (
-          <TeamMemberCard key={teamMember.name} {...teamMember} />
-        ))}
+const AboutPage: React.FC<PageProps> = () => {
+  const data = useStaticQuery(
+    graphql`
+      {
+        allFile(filter: { sourceInstanceName: { eq: "members" } }) {
+          nodes {
+            name
+            fields {
+              content
+            }
+          }
+        }
+      }
+    `,
+  );
+
+  const membersNames: string[] = [];
+  data.allFile.nodes.forEach((node: any) => {
```

Comment:
you could also do this in place and ensure that the value is never re-computed:
```
const membersNames: string[] = useMemo(() => data.allFile.nodes.reduce((arr, curr) => {
  arr.push(curr.fields.content);
  return arr;
}, [data.allFile.nodes]);
```

Reply:
I'm cool with replacing the forEach with reduce, but I think using useMemo is a little overkill. The size of the array should be relatively small, so I don't think we need to memoize the value.

Reply:
though it's fine as it is. I actually think that memoization might be useless here anyways, because `useMemo` only does shallow comparison, and since this site is static there isn't much re-computing happening anyways.

Reply:
> though it's fine as it is. I actually think that memoization might be useless here, because `useMemo` only does shallow comparison, and since this site is static there isn't much re-computing happening anyways.

Yeah that's a good point.